src/Umbraco.Tests/Testing/UmbracoTestBase.cs

```diff
@@ -228,6 +228,7 @@ protected virtual void ComposeWeb()
             Composition.RegisterUnique<IContentLastChanceFinder, TestLastChanceFinder>();
             Composition.RegisterUnique<IVariationContextAccessor, TestVariationContextAccessor>();
             Composition.RegisterUnique<IPublishedSnapshotAccessor, TestPublishedSnapshotAccessor>();
+            Composition.RegisterUnique<IIOHelper, IOHelper>();
```

Comment:
This will conflict with other merged PRs since IIOHelper is a 'essential' component that is treated in a slightly special way.  I'll update this PR to reflect the latest changes for this.